package.json

```diff
@@ -47,7 +47,7 @@
     "samples:build": "node ./tooling/index.js build --only-samples",
     "samples:serve": "webpack serve --color --env NODE_ENV=development",
     "samples:serve:integration": "webpack serve --color --env NODE_ENV=test",
-    "samples:test": "NODE_ENV=test wdio",
+    "samples:test": "NODE_ENV=test npx wdio run wdio.conf.js",
```

Comment:
I could see wdio available in devDependencies. Should we still be using npx here?

Reply:
locally it was causing issue so added that